admin/sql/updates/2019-07-09-add-recommendation-table.sql

```diff
@@ -0,0 +1,65 @@
+BEGIN;
+
+CREATE SCHEMA recommendation;
+
+CREATE TABLE recommendation.cf_recording (
+  id                  SERIAL, -- PK
+  user_id             INTEGER NOT NULL, --FK to "user".id
+  recording_msid      UUID NOT NULL,
+  type                cf_recording_type,
+  created             TIMESTAMP WITH TIME ZONE DEFAULT NOW()
```

Comment:
perhaps not null for this and the recommender table? What about `type` too?

Reply:
I wondered about this too, but I couldn't remember best practices. I'd like them and certainly on type. Good catch.